tests/instruments/keysight/test_keysightB2912A_with_device.py

```
@@ -0,0 +1,248 @@
+    assert keysightB2912A.ch2.current_measurement_speed == 1e-4
+
+
+def test_measured_current(keysightB2912A):
+    c = keysightB2912A.ch1.measured_current
+    assert isinstance(c, float)
+    c = keysightB2912A.ch2.measured_current
+    assert isinstance(c, float)
+
+
+def test_measured_current_array(keysightB2912A):
+    keysightB2912A.ch1.measured_current_array
```

Ruff (0.15.10)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)